File: app/src/main/java/cf/movie/slmovie/main/search/ui/ISearchActivity.java
package cf.movie.slmovie.main.search.ui;

import cf.movie.slmovie.main.search.bean.SearchAdapter;
import cf.movie.slmovie.main.search.event.SearchEvent;

/**
 * Created by 包俊 on 2017/8/12.
 */

public interface ISearchActivity {

    void setAdapter(SearchAdapter adapter);

    void reqError(String msg);

    void swipe(boolean refresh);
}

File: app/src/main/java/cf/movie/slmovie/main/search/ui/ISearchActivity.kt
package cf.movie.slmovie.main.search.ui

import cf.movie.slmovie.main.search.event.SearchEvent

/**
 * Created by 包俊 on 2017/8/12.
 */

interface ISearchActivity {

    fun setMovies(event: SearchEvent)

    fun reqError(msg: String)

    fun swipe(refresh: Boolean)
}
